lib/KeyboardAwareMixin.js

```diff
@@ -87,12 +87,14 @@ const KeyboardAwareMixin = {
     this.setState({keyboardSpace})
     // Reset scroll position after keyboard dismissal
     if (this.props.enableResetScrollToCoords === false) {
+      this.defaultResetScrollToCoords = null
       return
     } else if (this.resetCoords) {
       this.scrollToPosition(this.resetCoords.x, this.resetCoords.y, true)
     } else {
       if (this.defaultResetScrollToCoords) {
         this.scrollToPosition(this.defaultResetScrollToCoords.x, this.defaultResetScrollToCoords.y, true)
+        this.defaultResetScrollToCoords = null
```

Comment:
`defaultResetScrollToCoords ` will only be set if it's null. And the coords of the scrollview changes while scrolling. If I don't clear it here, it would be wrong after scrolling the scrollview.